extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -176,6 +177,19 @@ define(
 		$('.editpage-widemode-trigger').click(editorModeChange);
 	}
 
+	function launchEditorPreview(){
+		var editorValue = ace.getContent(),
+			wikiaDomain = mw.config.get('wgServer').split('://')[1],
+			templateName = (new mw.Title(mw.config.get('wgPageName'))).getMain(),
+			infoboxPreviewURL = mw.config.get('wgInfoboxPreviewURL'),
+			$form = $('<form>').attr({'action': infoboxPreviewURL, 'method': 'POST', 'target': '_blank'});
```

Comment:
it's just a readability / code separation optimisation but you could consider moving form creation to separate method